packages/db/fauna/resources/Function/findMetrics.js

```diff
@@ -0,0 +1,82 @@
+import fauna from 'faunadb'
+
+const {
+  Function,
+  CreateFunction,
+  Query,
+  Lambda,
+  Count,
+  Exists,
+  If,
+  Update,
+  Sum,
+  Match,
+  Index,
+  Map,
+  Get,
+  Var,
+  Select,
+  Documents,
+  Collection,
+  Paginate
+} = fauna
+
+const name = 'findMetrics'
+const body = Query(
+  Lambda(
+    [],
+    {
+      data: {
+        usersTotal: Count(Documents(Collection('User'))),
+        uploadsTotal: Count(Documents(Collection('Upload'))),
+        contentTotal: Count(Documents(Collection('Content'))),
+        contentTotalBytes: Sum(Match(Index('content_sizes'))),
+        dealsActiveTotal: Count(Match(
```

Comment:
We should be aware that `Count` and `Sum` work up to 100K items then we must paginate and page through.

Reply:
so, I can probably just create a Paginate Function that goes through all the pages.